cldrDict_sconscript

```diff
@@ -91,7 +91,6 @@ NVDAToCLDRLocales = {
 	"mn":("mn",),
 	#"mni":(),
 	"my":("my",),
-	"nb_NO":("nb",),
```

Comment:
This is not a right fix. Currently CLDR dictionary for Norwegian contains more than 3000 symbols and with your change Norwegian users of NVDA would be pretty negatively affected. Rather than just removing  problematic language please investigate why CLDR dictionary for Norwegian cannot be created and either fix this, or if it turns out to be a problem in CLDR report an issue against them.

Reply:
Norwegian is not in the CLDR repository. Therefore, there is a compilation problem. I can undo this fix if they add Norwegian again in the future.

Reply:
Please take a look at the migration section in http://cldr.unicode.org/index/downloads/cldr-39

Reply:
It should have been fixed by 06ad1ea.

Reply:
Thank you @lukaszgo1! I fixed it.